packages/bpk-component-calendar-scrollable/readme.md

```diff
@@ -0,0 +1,24 @@
+# bpk-component-calendar-scrollable
+
+> Backpack example component.
```

Comment:
Should be `Backpack scrollable calendar component.` (or whatever name we end up on).